tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -36,15 +36,15 @@
 
     <!-- Postgres SQL .NET framework references -->
     <PackageReference Include="npgsql" Version="4.0.0" Condition="'$(TargetFramework)' == 'net462'" />
-    <PackageReference Include="npgsql" Version="4.0.5" Condition="'$(TargetFramework)' == 'net471'" />
-    <PackageReference Include="npgsql" Version="4.0.10" Condition="'$(TargetFramework)' == 'net48'" />
-    <PackageReference Include="npgsql" Version="4.0.13" Condition="'$(TargetFramework)' == 'net481'" />
+    <PackageReference Include="npgsql" Version="4.0.13" Condition="'$(TargetFramework)' == 'net471'" />
```

Comment:
Since the instrumentation for npgsql now has three distinct version ranges (<4.1, >4.1 && <5.0, >5.0), would it make sense to use the `net471` TFM to test a version between 4.1 and 5.0, like you're using netcoreapp3.1 to test on the core agent side?

Reply:
Seems reasonable enough. I'll make it so!